lms/templates/lti_provider/user-auth-error.html

```diff
@@ -0,0 +1,24 @@
+<%page expression_filter="h" />
+<%namespace name='static' file='../static_content.html'/>
+<%!
+from django.utils.translation import gettext as _
+from openedx.core.djangolib.markup import HTML, Text
+%>
+<%inherit file="../main.html" />
+
+<section class="outside-app">
+  <h1>
+    ${Text(_("There was an error when loading this module!"))}
+  </h1>
+  <p>
+    ${Text(_("This module is available only for users who are signed into {platform}. "
+        "Kindly sign in and refresh this page to load the content."
```

Comment:
We should mention that the account needs to be associated with the same email on both platforms, right? Or do we want to allow linking the LTI user to the one who is currently logged into Open edX?

Reply:
@Agrendalath Right now the whole linking depends on the email being sent in the LTI launch request (`lis_person_contact_email_primary`). So keeping it restricted to allow this only when emails match might be the best. While not intuitive, we can also avoid scenarios like shared computers in college libraries where the new LTI user gets connected to the previous edX user.

Reply:
Update the error page in 0642dd7a8bab93500757de467bd3ea4bb6c6ae9f